### PR TITLE
Ui profile banner components + snapshot tests

### PR DESCRIPTION
**Pull request by bujars**
This includes the Profile Banner: Avatar, Username, Following, Followers, and Posts Buttons.

Only Snapshot Tests have been created + Passed

and Unit Tests are in process.

### CODE REVIEW

File: __tests__/FollowingButton-test.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      'This is the following list of ' + user_id,
+    );
+  });
+}
+
+/** Check that the property values passed into the prop, appear on the prop. */
+function testProperPropsPassed(user_id, following_count) {
+  it('User ID value is the Following Button prop value', () => {
+    /** Does it matter if we pass in follower_count simulatanously? */
+    const wrapper = shallow(<FollowingButton user_id={user_id} />);
```
Comment (kartikeya01) [inline __tests__/FollowingButton-test.js:35]: Are we handling an empty user_id?

File: __tests__/FollowerButton-test.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    /** Does it matter if we pass in follower_count simulatanously? */
+    const wrapper = shallow(<FollowerButton user_id={user_id} />);
+    const instance = wrapper.instance();
+    const userIDProp = instance.props.user_id;
+    expect(user_id).toEqual(userIDProp);
+  });
+
+  it('Follower Count value is the FollowerButton prop value', () => {
+    /** Does it matter if we pass in follower_count simulatanously? */
+    const wrapper = shallow(<FollowerButton follower_count={follower_count} />);
```
Comment (kartikeya01) [inline __tests__/FollowerButton-test.js:48]: What if this is a negative value? Does anything in the UI change? We should make sure min is 0

Comment (bujars) [reply __tests__/FollowerButton-test.js:48]: @kartikeya01  The view will be 0, as seen in testChipInnerText. However, to keep the component pure we can't change the actual prop value. If we wanted to pass that prop value over, we can use the 0 declared as the variable

File: __tests__/FollowingButton-test.js (hunk 1 above)
Comment (bujars) [reply __tests__/FollowingButton-test.js:35]: We set prop types which warns us if a string is not entered

File: __tests__/FollowerButton-test.js (hunk 2 above)
Comment (bujars) [reply __tests__/FollowerButton-test.js:48]: that being noted, a warning is development is noted if a nonnegative integer prop is passed in

File: App.js [hunk 3]
```diff
@@ -1,53 +1,17 @@
...
-          options={{tabBarLabel: 'Profile'}}
-        />
-        <Tab.Screen
-          name="Search"
-          component={SearchScreen}
-          options={{tabBarLabel: 'Search'}}
-        />
-      </Tab.Navigator>
-    </NavigationContainer>
+    <ProfileScreen />
```
Comment (bujars) [inline App.js:14]: Just removed everything on my screen for dev. Ignore this page

**Pull request closed without merging**